components/feature/downloads/src/main/res/layout/mozac_downloads_prompt.xml

```diff
@@ -64,12 +64,13 @@
         android:layout_alignBaseline="@id/icon"
         android:layout_marginStart="3dp"
         android:layout_marginTop="16dp"
+        android:layout_marginEnd="16dp"
```

Comment:
Should we give padding evenly start/end `16dp`? As now it seems that we are giving `layout_marginStart` of `3dp` and `layout_marginEnd` of `16dp`?

Reply:
I thought the same when I initially saw the layout! Surprisingly for me, it uses `android:layout_toEndOf="@id/icon"` to move it away from the start, and 3 extra margin from the icon.